jwst/tso_photometry/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Find TSO photometry sources and save them in a catalog."""
```

Comment:
The step doesn't actually do any source finding, as it assumes that there's a single source at the reference position.  I'd suggest updating this to something like, "Perform aperture photometry for a time-series observation"